src/register/register-gnome/gnucash-item-edit.c

```diff
@@ -167,8 +167,7 @@ gnc_item_edit_init (GncItemEdit *item_edit)
 
     item_edit->popup_toggle.ebox = NULL;
     item_edit->popup_toggle.tbutton = NULL;
-    item_edit->popup_toggle.arrow_up = NULL;
-    item_edit->popup_toggle.arrow_down = NULL;
+    item_edit->popup_toggle.direction = TRUE;
```

Comment:
This is a variablename that doesn't suggest a boolean value. Better would be something like arrow_points_up or something like that.
Other than that I very much appreciate this change. Much cleaner.